android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -12,6 +12,8 @@
 import android.os.Bundle;
 
 public class TagDispatcher {
+    private static final int DELAY_PRESENCE = 60000;
```

Comment:
@slomo @petter-fidesmo  Is a presence check even applicable to NFC A tags used in IsoDep mode? 

I'm in favour of disabling the presence check completely...